app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,2 @@
+class PostsController < ApplicationController
+end
```

Comment:
- [ ] It would be nice if you make methods `index`, and `show` into your post controller because these methods are responsible for redirecting the UI and for some logical operations 🤔 So kindly add index and show methods in your posts controller 👍 to follow the project requirements 😊